openslides/utils/auth.py

```diff
@@ -46,6 +47,18 @@ def anonymous_is_enabled() -> bool:
     return config['general_system_enable_anonymous']
 
 
+async def a_anonymous_is_enabled() -> bool:
```

Comment:
Please rename to something like `async_anonymous_is_enabled()`?

Reply:
ok. I thought the prefix a was common for an async method in python, see https://www.python.org/dev/peps/pep-0525/#aiter-and-anext-builtins 

But I changed it now.